data/gegno/gegno intro missions.txt

```diff
@@ -0,0 +1,553 @@
+			`You arrive once more at Giaru Gegno, hoping to find another Quarg that speaks your language. This time, however, you notice that a few of the Gegno on this ringworld occasionally give you a look of interest as they walk by. After about a minute of this a pair of Quarg approach you, motioning you to follow them. Led back to the familiar room from before, the Quarg you previously spoke with is already calmly sitting at the other end of the table.`
+			`	"You have astonishingly achieved a wonderous feat, brave traverser," the Quarg says, placing its hands together. It seems the Quarg are already aware of your encounter on the alien homeworld. In fact, thinking back to it, the looks of interest you received earlier suggests that the word spread quickly. "Not significantly often it is the mighty Gegno interact with daring outsiders."`
+			choice
+				`	"It was a coincidence."`
+				`	"They seemed in trouble when I found them, so I helped them."`
+					goto "trouble"
+			`	"Indeed yes, strange coincidences do curiously happen. Many coincidences are as mere fleeting moments of subtle peculiarity." The Quarg then displays a soft grimace. "Sometimes, one may be grand and powerful, in addition to potent consequences. The Gegno have learned this a great number of times.`
+			label "trouble"
+			`	"Quite a brave one you indeed are. Most would think to not leap into the daunting face of uncertainty as you have." The Quarg momentarily meditates on its next thought. "Though, as the Gegno have throughout their history, some may make monumentous jumps undeterred by any amount of hesitance."`
+			`	The Quarg raises from its seat, and you act in accordance. "Please do be distinctly cautious in your encounters with the Gegno. Always feel gracious to visit here once more, should you mindfully chose to do so." As you are directed out of the room, you turn back to see the Quarg bowing. You can't help but wonder what was on their mind during the conversation, but it might be for the best you are more careful from now on.`
+				decline
+
+
+
+# Gegno Superheavy
+
+mission "Stationed Augen"
+	invisible
+	landing
+	npc save
+		government "Gegno Vi"
+		personality heroic vindictive staying
+		system "Kiluit"
+		ship "Augen" "Vytii Yrrlausei"
+	to fail
+		has "event: Vesvii war"
+
+mission "Augen Destruction"
+	landing
+	invisible
+	to offer
+		has "Stationed Augen: failed"
+		not "event: Vesvii war"
+	on offer
+		set "dead augen"
+		fail
+
+mission "Active Augen"
+	invisible
+	to offer
+		has "event: Vesvii war"
+		has "Stationed Augen: failed"
+		not "dead augen"
+	npc save
+		government "Gegno Vi"
+		personality heroic vindictive staying
+		system "Vesvii"
+		ship "Augen" "Vytii Yrrlausei"
+
+event "Vesvii War"
+
+
+
+# Gegno Defense trigger
+
+mission "Gegno Genocide Defense"
+	landing
+	repeat
+	invisible
+	to offer
+		not "Gegno Genocide Defense: active"
+		or
+			"reputation: Gegno" < -100
+			"reputation: Gegno Vi" < -100
+			"reputation: Gegno Scin" < -100
```

Comment:
The conversations suggests you're unwanted now because you attacked civvies, yet this can offer even if you never attacked the main Gegno government.